src/index.html

```diff
@@ -8,6 +8,49 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Stars</h1>
+    <div>
+      <div class="stars--0 stars">
+        <span class="stars__star"></span>
+        <span class="stars__star"></span>
+        <span class="stars__star"></span>
+        <span class="stars__star"></span>
+        <span class="stars__star"></span>
+      </div>
+      <div class="stars--1 stars">
+        <span class="stars__star star--active"></span>
+        <span class="stars__star"></span>
+        <span class="stars__star"></span>
+        <span class="stars__star"></span>
+        <span class="stars__star"></span>
+      </div>
+      <div class="stars--2 stars">
+        <span class="stars__star star--active"></span>
```

Comment:
remove `star--active` from all stars, make them active using stars--1, stars--2, stars--3 ... classes
you should make a selector with all these classes is CSS

hint: use `nth-child` in css